lib/services/elasticsearch.js

```diff
@@ -885,26 +893,43 @@ class ElasticSearch extends Service {
           errors: []
         };
 
-        let row;
+        let
+          i = 0,
+          row;
+
 
         while ((row = body.items.shift()) !== undefined) {
           const
             action = Object.keys(row)[0],
             item = row[action];
 
           if (item.status >= 400) {
-            item.action = action;
+            const error = {
+              status: item.status,
+              _id: item._id,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this trailing comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ATrailingComma)